core/root.go

```diff
@@ -27,7 +27,7 @@ type ABRootManager struct {
 
 // ABRootPartition represents an ABRoot partition
 type ABRootPartition struct {
-	Label        string // a,b
+	Label        string // vos-a, vos-b
```

Comment:
ABRoot should be Distro Agnostic.

Reply:
This is more of an example. Label gets whatever is in `partLabelA` and `partLabelB` inside `abroot.json`

Reply:
I'll try to be more explicit about that

Reply:
This is an example